scripts/src/ui/page/earnThankYou.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import BasePage from "./base";
+
+import "../../../../styles/src/page/earnThankYou.styl";
+
+export interface EarnThankYouProps {
+	amount: number;
+}
+
+const getImageUrl = (imageName: string) => "https://s3.amazonaws.com/htmlpoll.kinecosystem.com/images/${imageName}.svg".replace("${imageName}", imageName);
+
+export class EarnThankYou extends React.Component<any, EarnThankYouProps> {
+	public render() {
+		return (
+			<BasePage className={"earnThankYou"}>
+
+				<div className={"text"}>
+					<div>Yay! you've earned</div>
+					<div className={"amount"}>{this.props.amount}</div>
+					<img src={getImageUrl("kin-type")} className={"kin-type"}/>
+				</div>
+				<div className={"footer"}>
+					<img src={getImageUrl("coins")} className={"coins"}/>
+					<img src={getImageUrl("sparks")} className={"sparks"}/>
+					<img src={getImageUrl("diamond-l-1")} className={"diamond-l-1 diamonds"}/>
```

Comment:
instead of all of that, I'd do:
```
// at the top of the function:
const diamonds = ["l-1", "l-2", "l-3", "l-4", "c-1", "c-2", "r-3", "r-2", "r-1"].map(name => <img src={ getImageUrl("diamond-" + name ) } className={"diamond-" + name + " diamonds"}/>);
```

and then:
```
<div className={"footer"}>
	<img src={getImageUrl("coins")} className={"coins"}/>
	<img src={getImageUrl("sparks")} className={"sparks"}/>
        { diamonds }
</div>
```